book/5-data-mgmt-exp-proc/eda-cv-tasks.md

```diff
@@ -0,0 +1,117 @@
+# What are some of the ways to do EDA for CV tasks?
+
+Contributor: Calvin Neo, AI Engineer
+
+---
+
+The purpose of this article is to guide new AI Engineers on performing Exploratory Data Analysis of Computer Vision tasks in a systematic manner. It assumes a basic understanding of image format, and how bounding boxes and masks are drawn.
+
+## Image EDA  
+
+### Basic Image Analysis  
+The basic image analysis considers the broad characteristics of the images that have been obtained.
+
+The guiding questions are as such:   
+1. What is the number of images provided? Is this the number of images promised to be delivered for experiments?
+1. What is the format of images eg JPG/PNG/JPEG? If there are multiple formats given, what are the counts for each image format? 
+1. What are the image sizes/resolution (eg 1080x960)? If there are multiple dimensions, what are the counts for each dimension?
+1. How similar are the images to each other? You can do an EDA to check on this by performing image hashing to discover groups which are similar to each other.
```

Comment:
Suggest adding this to point 4:
`This also helps to uncover whether some of the images came from a continuous video stream of a scene that is changing slowly - in which case, these images may be considered near-duplicates.`